otter/test/models/test_intents.py

```diff
@@ -15,6 +19,32 @@ def test_perform(self):
                                modifier=lambda g, o: 'new state')
         dispatcher = TypeDispatcher({
```

Comment:
I know this part wasn't really changed, but would it make sense to just use the `get_cassandra_dispatcher` here, so that it tests that `get_cassandra_dispatcher` can get performers for `ModifyGroupState`?